tgui/packages/tgui/interfaces/Rockbox.js

```diff
@@ -0,0 +1,87 @@
+
+import { useBackend, useSharedState } from '../backend';
+import { Button, LabeledList, Section, NumberInput, Box, Divider, Tooltip } from '../components';
+import { Window } from '../layouts';
+
+export const Rockbox = (props, context) => {
+  const { act, data }=useBackend(context);
+  const {
+    name,
+    amount,
+    forSale,
+    price,
+    stats,
+  }=data;
+  const [takeAmount, setTakeAmount] = useSharedState(context, 'takeAmount', 1);
+  return (
+
+    <Window
+      title="Rockbox"
+      width={500}
+      height={500}
+      scrollable
+    >
+      <Window.Content>
+        <Section>
+          <Box>{"Take Amount: "}
+            <NumberInput
+              value={takeAmount}
+              width={4}
+              minValue={1}
+              onDrag={(e, value) => setTakeAmount(value)}
+            />
+          </Box>
+        </Section>
+        <Section>
+          {data.ores.length ? (
+            <LabeledList>
+              {data.ores.map((currentOre) => (
+                <>
```

Comment:
You'll have to add a key property here to let inferno/react know which item this is is in the list for more performant rendering. I'm pretty sure you can write something like `<Fragment key={currentOre...}>` for fragments